### DISCUSSION TITLE
discuciones

### DISCUSSION THREAD

**Discussion by saul19bautista** [SistemasTecTlaxiaco/reciclanexus#6] [Announcements]
¿qué hiciste en la ultima sesión?  @Rosa218 , @Luz1727 

¿qué haré en esta sesión? @Rosa218, @Luz1727 

¿qué obstáculos tengo? @Rosa218 , @Luz1727

**Comment by Rosa218** [comment]
En la ultima sesión se realizo el sprint planing, creando las incidencias y asignarlas a kanban.

En esta sesión se consultara la información para implementar el contador

El obstáculo que se tiene hasta el momento, son algunas extensiones que nos pide para la implementación del contador, las cuales se necesitan instalar.
